Client_OM_Sample/ClientOmConsoleApp/ClientOmConsoleApp/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using Microsoft.VisualStudio.Services.Client;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+
+namespace ClienOmConsoleApp
+{
+    class ClientOmConsoleApp
+    {
+        //Console application to execute a user defined work item query
+        static void Main(string[] args)
+        {
+            //=========== Config [edit these] ===================
+            string collectionUri = "https://peakyyy.visualstudio.com";
+            Wiql query = new Wiql() { Query = "Select [State], [Title] from WorkItems where [Work Item Type] = 'Bug' And [Tags] Contains 'findMe'" };
+            //===================================================
+
+            //Prmot user for credential for collection specified above
+            VssCredentials cred = new VssClientCredentials(false);
+            cred.PromptType = CredentialPromptType.PromptIfNeeded;
```

Comment:
I couldn't find any, I just tried each one until the sample worked to be honest.